artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageSubscriptionImpl.java

```diff
@@ -832,13 +832,14 @@ public boolean isPersistent() {
    @Override
    public void processReload() throws Exception {
       if (recoveredACK != null) {
-         logger.trace("********** processing reload!!!!!!!");
+         logger.debug("processing reload queue name={} with id={}", queue != null ? this.queue.getName() : "N/A", this.cursorId);
```

Comment:
The cursorId value is a long, so you should now gate the logger to avoid boxing. The "this" qualifiers dont seem needed.